### DISCUSSION TITLE
Policies and notifications

### DISCUSSION THREAD

**Discussion by roadSurfer** [DependencyTrack/dependency-track#1490] [Q&A] [answered]
I've just started playing around with DT 4.4. I have managed to set basic policies, but I am just wondering about a few things.

1. Are policies only assessed on BoM upload?
    * If I add/update a policy then no reassessment of projects occurs, or is there an API endpoint I can hit from a cronjob or similar?
3. It is not possible to notify on a new policy failure?
    * Notifying on each new vulnerability (or vulnerable dependency) leads to a lot of ntoficiations, even for a small project, I'd rather notify on a policy being failed. Maybe I am just not seeing the setting.
    * On the topic of notifications, can I modify the template to link back to the project? The Teams notifications don't seem to include a URL.

Thanks!

**Comment by stevespringett** [comment, marked as answer]
Policies are evaluated whenever components are analyzed for vulnerabilities. This occurs on upload and daily (if no boms are uploaded). There was an oversight for portfolio analysis that was resolved in #1492 and will be included in v4.5.

Notifications on policy is currently scheduled for inclusion in v4.5 via #1396.

**Reply by roadSurfer** [reply]
Ah, great! Thanks for getting back to me.
